models/intermediate/int_klaviyo__event_attribution.sql

```diff
@@ -108,11 +110,10 @@ final as (
         *,
 
         -- get whether the event is attributed to a flow or campaign
-        case when last_touch_id is not null then 
-            coalesce(touch_type, first_value(touch_type) over(
-                partition by person_id, touch_session order by occurred_at asc rows between unbounded preceding and current row)) 
+        coalesce(touch_type, first_value(touch_type) over(
```

Comment:
What is your understanding on what was wrong with the original code?  

I know we talked about the possibility of not being able to use first_value, but it looks like we can.